ee_plugin/utils.py

```diff
@@ -60,25 +60,25 @@ def get_layer_by_name(name):
     return None
 
 
-def add_or_update_ee_image_layer(image, name, visibility=True, opacity=1.0):
+def add_or_update_ee_image_layer(image, name, shown=True, opacity=1.0):
     layer = get_layer_by_name(name)
 
     if layer: 
         if not layer.customProperty('ee-layer'):
             raise Exception('Layer is not an EE layer: ' + name)
     
-        update_ee_image_layer(image, layer, visibility, opacity)
+        update_ee_image_layer(image, layer, shown, opacity)
     else:
-        add_ee_image_layer(image, name, visibility, opacity)
+        add_ee_image_layer(image, name, shown, opacity)
 
 
-def add_ee_catalog_image(name, asset_name, vis_props, collection_props):
+def add_ee_catalog_image(name, asset_name, visParams, collection_props):
```

Comment:
... camel notation is not PEP8 enough

Reply:
yes of course I know it is not PEP8 rule but I try to follow the params name used by the addLayer function (to maintain consistency): https://developers.google.com/earth-engine/api_docs#map.addlayer